recipes/default.rb

```diff
@@ -85,19 +86,17 @@
   end
 
   yum_repository 'webtatic' do
-    description (node['librenms']['repo_webtatic']['desc']).to_s
-    baseurl (node['librenms']['repo_webtatic']['url']).to_s
+    description node['librenms']['repo_webtatic']['desc']
+    baseurl node['librenms']['repo_webtatic']['url']
     gpgcheck false
-    enabled true
-    only_if { node.normal['librenms']['repo_webtatic']['enabled'] = 'true' }
+    enabled node['librenms']['repo_webtatic']['enabled']
   end
 
   yum_repository 'epel' do
-    description (node['librenms']['repo_epel']['desc']).to_s
-    baseurl (node['librenms']['repo_epel']['url']).to_s
+    description node['librenms']['repo_epel']['desc']
+    baseurl node['librenms']['repo_epel']['url']
     gpgcheck false
-    enabled true
-    only_if { node.normal['librenms']['repo_epel']['enabled'] = 'true' }
+    enabled node['librenms']['repo_epel']['enabled']
```

Comment:
I still don't understand why you are not using `yum-epel`

Reply:
Fixed.